source/MaterialXView/ShaderValidators/Glsl/GlslValidator.cpp

```diff
@@ -230,18 +233,67 @@ bool GlslValidator::createTarget()
     glFramebufferTexture2D(GL_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_TEXTURE_2D, _depthTarget, 0);
     
     glBindTexture(GL_TEXTURE_2D, UNDEFINED_OPENGL_RESOURCE_ID);
-
     glDrawBuffer(GL_NONE);
 
-    // Validate the framebuffer
+    // Validate the framebuffer. Default to fixed point if we cannot get
+    // a floating point buffer.
     GLenum status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
     if (status != GL_FRAMEBUFFER_COMPLETE)
+    {
+        glDeleteTextures(1, &_colorTarget);
```

Comment:
Fallback case added to use fixed point buffer as float can fail. We'll may end up with banding w/o a float buffer for color management, but better to have a buffer than not.